crates/core/src/metadata_store.rs

```diff
@@ -26,6 +32,7 @@ use restate_types::storage::{StorageCodec, StorageDecode, StorageEncode, Storage
 use restate_types::{flexbuffers_storage_encode_decode, Version, Versioned};
 use std::future::Future;
 use std::sync::Arc;
+use std::time::Instant;
```

Comment:
In general, it's better to use tokio's Instant, but that's not an issue in this case. Just a reminder to form the habit.